### DISCUSSION TITLE
Export my Custom Categories from AdiBags to BatterBags?

### DISCUSSION THREAD

**Discussion by AndrewSimpson2022** [AdiAddons/AdiBags#1000] [Ideas]
Is there anyway to export my Custom Categories from AdiBags to BetterBags?

I've got years worth of Custom Categories, e.g. Shadow - Herbs, Pand - Herbs, Dragon - Herbs.

Any export/import feature planned?

(Thanks for the great Add-On)

**Comment by SpareSimian** [comment]
I asked about this way back when BetterBags was started and was told no conversion would be available. So I've been patiently waiting for BB to become reasonably stable enough for me to switch and then I'd write my own conversion utility.

**Comment by Cidan** [comment]
There will be no export from AdiBags, but there will be an import feature in BetterBags. What you could do is read your SavedVariables config and eventually dump it (in the right format) to BetterBags. I'll do my best to make this as easy as possible.

**Comment by AndrewSimpson2022** [comment]
Thanks, I look forward to moving to BetterBags.
